2022-02-02 03:37:45,020 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-02 03:37:45,022 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /work/wandb/settings
2022-02-02 03:37:45,022 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 03:37:45,022 INFO    MainThread:71 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-02 03:37:45,023 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:37:45,023 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:37:45,023 INFO    MainThread:71 [wandb_init.py:_log_setup():386] Logging user logs to /work/wandb/run-20220202_033744-xgb_depth_test/logs/debug.log
2022-02-02 03:37:45,023 INFO    MainThread:71 [wandb_init.py:_log_setup():387] Logging internal logs to /work/wandb/run-20220202_033744-xgb_depth_test/logs/debug-internal.log
2022-02-02 03:37:45,024 INFO    MainThread:71 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f1ad1e79190>
2022-02-02 03:37:45,024 INFO    MainThread:71 [wandb_init.py:init():420] calling init triggers
2022-02-02 03:37:45,025 INFO    MainThread:71 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-02-02 03:37:45,026 INFO    MainThread:71 [wandb_init.py:init():442] re-initializing run, found existing run on stack: xgb_depth_test
2022-02-02 03:37:45,039 INFO    MainThread:71 [wandb_run.py:finish():1468] finishing run cz4041/test-project/xgb_depth_test
2022-02-02 03:37:45,039 INFO    MainThread:71 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-02 03:37:45,042 INFO    MainThread:71 [wandb_run.py:_restore():1769] restore
2022-02-02 03:37:45,828 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:37:45,956 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:37:46,174 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:37:46,283 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 72700
}

2022-02-02 03:37:46,422 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 72700
}

2022-02-02 03:37:46,524 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}

2022-02-02 03:37:46,626 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}

2022-02-02 03:37:46,728 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}

2022-02-02 03:37:46,831 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}

2022-02-02 03:37:46,933 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}

2022-02-02 03:37:47,139 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}

2022-02-02 03:37:47,393 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 72700
  total_bytes: 72700
}
local_info {
}

2022-02-02 03:37:49,000 INFO    MainThread:71 [wandb_run.py:_append_files():2194] logging synced files
2022-02-02 03:37:49,062 INFO    MainThread:71 [wandb_init.py:init():471] starting backend
2022-02-02 03:37:49,062 INFO    MainThread:71 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-02 03:37:49,067 INFO    MainThread:71 [backend.py:ensure_launched():219] starting backend process...
2022-02-02 03:37:49,095 INFO    MainThread:71 [backend.py:ensure_launched():225] started backend process with pid: 1917
2022-02-02 03:37:49,098 INFO    MainThread:71 [wandb_init.py:init():480] backend started and connected
2022-02-02 03:37:49,119 INFO    MainThread:71 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-02-02 03:37:49,119 INFO    MainThread:71 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-02 03:37:49,119 INFO    MainThread:71 [wandb_init.py:init():550] updated telemetry
2022-02-02 03:37:49,132 INFO    MainThread:71 [wandb_init.py:init():581] communicating current version
2022-02-02 03:37:53,322 INFO    MainThread:71 [wandb_init.py:init():586] got version response 
2022-02-02 03:37:53,322 INFO    MainThread:71 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-02-02 03:37:53,434 INFO    MainThread:71 [wandb_init.py:init():624] starting run threads in backend
2022-02-02 03:37:58,449 INFO    MainThread:71 [wandb_run.py:_console_start():1827] atexit reg
2022-02-02 03:37:58,455 INFO    MainThread:71 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-02-02 03:37:58,455 INFO    MainThread:71 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-02-02 03:37:58,456 INFO    MainThread:71 [wandb_run.py:_redirect():1762] Redirects installed.
2022-02-02 03:37:58,457 INFO    MainThread:71 [wandb_init.py:init():651] run started, returning control to user process
2022-02-02 03:37:58,462 INFO    MainThread:71 [wandb_run.py:_config_callback():966] config_cb None None {'boosting_type': 'dart', 'objective': 'multi:softmax', 'num_class': 9, 'max_depth': 2, 'min_child_weight': 20, 'gamma': 1, 'subsample': 0.8, 'colsample_bytree': 0.7, 'tree_method': 'hist', 'eval_metric': 'mlogloss', 'eta': 0.04, 'alpha': 1, 'verbose': 2}
2022-02-02 03:38:43,249 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:38:43,254 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:38:43,288 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:38:43,492 INFO    MainThread:71 [wandb_init.py:_pause_backend():299] pausing backend
2022-02-02 03:38:43,492 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:53:44,658 INFO    MainThread:71 [wandb_init.py:_resume_backend():308] resuming backend
2022-02-02 03:53:44,694 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /root/.config/wandb/settings
2022-02-02 03:53:44,694 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from /work/wandb/settings
2022-02-02 03:53:44,694 INFO    MainThread:71 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-02-02 03:53:44,694 INFO    MainThread:71 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-02-02 03:53:44,694 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:53:44,695 INFO    MainThread:71 [wandb_setup.py:_flush():75] Applying login settings: {'api_key': '***REDACTED***'}
2022-02-02 03:53:44,695 INFO    MainThread:71 [wandb_init.py:_log_setup():386] Logging user logs to /work/wandb/run-20220202_035344-xgb_depth_test/logs/debug.log
2022-02-02 03:53:44,695 INFO    MainThread:71 [wandb_init.py:_log_setup():387] Logging internal logs to /work/wandb/run-20220202_035344-xgb_depth_test/logs/debug-internal.log
2022-02-02 03:53:44,695 INFO    MainThread:71 [wandb_init.py:init():420] calling init triggers
2022-02-02 03:53:44,695 INFO    MainThread:71 [wandb_init.py:init():427] wandb.init called with sweep_config: {}
config: {}
2022-02-02 03:53:44,696 INFO    MainThread:71 [wandb_init.py:init():442] re-initializing run, found existing run on stack: xgb_depth_test
2022-02-02 03:53:44,702 INFO    MainThread:71 [wandb_run.py:finish():1468] finishing run cz4041/test-project/xgb_depth_test
2022-02-02 03:53:44,723 INFO    MainThread:71 [jupyter.py:save_history():458] saving 27 cells to _session_history.ipynb
2022-02-02 03:53:44,752 INFO    MainThread:71 [wandb_run.py:_config_callback():966] config_cb ('_wandb', 'session_history') code/_session_history.ipynb None
2022-02-02 03:53:44,841 INFO    MainThread:71 [jupyter.py:save_ipynb():377] looking for notebook: None
2022-02-02 03:53:44,842 INFO    MainThread:71 [wandb_init.py:_jupyter_teardown():318] cleaning up jupyter logic
2022-02-02 03:53:44,842 INFO    MainThread:71 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-02-02 03:53:44,844 INFO    MainThread:71 [wandb_run.py:_restore():1769] restore
2022-02-02 03:53:45,361 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:53:45,541 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 571
}

2022-02-02 03:53:46,438 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 82127
}

2022-02-02 03:53:46,542 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 82127
}

2022-02-02 03:53:46,677 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 571
  total_bytes: 82127
}

2022-02-02 03:53:46,780 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}

2022-02-02 03:53:46,888 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}

2022-02-02 03:53:46,993 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}

2022-02-02 03:53:47,095 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}

2022-02-02 03:53:47,210 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}

2022-02-02 03:53:47,444 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}

2022-02-02 03:53:47,687 INFO    MainThread:71 [wandb_run.py:_wait_for_finish():1929] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 82127
  total_bytes: 82127
}
local_info {
}

2022-02-02 03:53:49,400 INFO    MainThread:71 [wandb_run.py:_append_files():2194] logging synced files
